README.rst

```diff
@@ -40,6 +40,17 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user:
```

Comment:
With a separate PR I think we should mention earlier in README file that the formula doesn't support deploying InfluxDB in cluster. And then we can remove saying "Single-node" here and elsewhere.